python/pyspark/sql/functions.py

```diff
@@ -2519,19 +2519,25 @@ def to_utc_timestamp(timestamp: "ColumnOrName", tz: "ColumnOrName") -> Column:
 
 def timestamp_seconds(col: "ColumnOrName") -> Column:
     """
+    Converts the number of seconds from the Unix epoch (1970-01-01T00:00:00Z)
+    to a timestamp.
+
     .. versionadded:: 3.1.0
 
     Examples
     --------
     >>> from pyspark.sql.functions import timestamp_seconds
-    >>> spark.conf.set("spark.sql.session.timeZone", "America/Los_Angeles")
+    >>> spark.conf.set("spark.sql.session.timeZone", "UTC")
```

Comment:
Why do you need to change the time zone here? It impacts on `show()` actually, and it doesn't matter, I think.

Reply:
I find that specifying a particular time zone might insiunate that the result depends on the set time zone, while it in fact only affects the `.show()`. No strong opinion here though, happy to revert if you want to keep the changes to a minimum.